qa/suites/rgw/multisite/realms/three-zone.yaml

```diff
@@ -18,3 +18,4 @@ overrides:
             endpoints: [c2.client.0]
           - name: test-zone3
             endpoints: [c1.client.1]
+            is_pubsub: true
```

Comment:
it's important that we still test replication over 3 zones, because it exposes an extra kind of race that we need to be able to handle (ie zone 3 syncing from an object from zone 1 before/after zone 2)

in multisite/clusters.yaml, we deployed an extra `c2.client.1` that is unused in this `three-zone.yaml` case, so i suggested adding it here as a 4th zone for pubsub, ie
```diff
           - name: test-zone3
             endpoints: [c1.client.1]
+          - name: test-zone4
+            endpoints: [c2.client.1]
             is_pubsub: true
```

Reply:
should probably rename the file to four-zones.yaml? or maybe create a new suite for pubsub?

Reply:
splitting pubsub out of three-zones.yaml could make sense. the tradeoff here is between making the existing three-zone.yaml jobs take longer, vs. adding two extra jobs to the suite (a pubsub.yaml here would get run against both frontend/beast.yaml and frontend/civetweb.yaml)

since the pubsub tests take <10min and the majority of the job runtime is in setup/teardown, i think it's best to combine pubsub in three-zones.yaml. maybe just rename it to something like three-zones-plus-pubsub.yaml?